### PR TITLE
Feature challenge library

### PR DESCRIPTION
**Pull request by masumomo**

### CODE REVIEW

File: components/CreateChallenge/Form02ChallengeType.tsx [hunk 1]
```diff
@@ -78,15 +78,15 @@ function Form02ChallengeType({navigation}, props) {
...
+          <CardItem onPress={() => navigation.navigate('ChallengeLibraryInfo')}>
             <Grid style={styles.Grid}>
               <Col style={{width: 50}}>
                 <Icon
                   // active
                   // style={styles.QuantIcon}
                   name="refresh"
                   style={styles.Icon}
-                  onPress={() => navigation.navigate('ChallengeRepeatInfo')}
+                  onPress={() => navigation.navigate('ChallengeLibraryInfo')}
```
Comment (travisricks) [inline components/CreateChallenge/Form02ChallengeType.tsx]: @masumomo Why are you removing `ChallengeRepeatInfo`?

File: components/CreateChallenge/Form02ChallengeType.tsx [hunk 2]
```diff
@@ -104,6 +104,21 @@ function Form02ChallengeType({navigation}, props) {
...
+            <Text style={{paddingLeft: 120}}>Search Challenge</Text>
+          </CardItem>
+          <CardItem
+            button
+            onPress={() => navigation.navigate('ChallengeLibrary')}>
+            <Body>
+              <Text>Pick challenge that someone created.</Text>
+            </Body>
+          </CardItem>
+        </Card>
```
Comment (travisricks) [inline components/CreateChallenge/Form02ChallengeType.tsx]: @masumomo Please follow the style I implemented above, for example Lines 54-79. You can try `library` for icon name

File: components/CreateChallenge/Form03ChallengeRepeat.tsx [hunk 3]
```diff
@@ -35,12 +35,16 @@ function Form03ChallengeRepeat({navigation, route}, props) {
...
+          onChangeText={TextInputValue => {
             state.setChallengeInput({
               ...state.challengeInput,
               taskName: TextInputValue,
-            })
-          }
+            });
+            console.log(
+              'state.challengeInput.taskName :',
+              state.challengeInput.taskName,
```
Comment (travisricks) [inline components/CreateChallenge/Form03ChallengeRepeat.tsx]: please remove console.log

File: components/CreateChallenge/FormGroup02ChallengeType.tsx [hunk 4]
```diff
@@ -119,6 +119,25 @@ function FormGroup02ChallengeType({navigation}, props) {
             </Grid>
           </CardItem>
         </Card>
+
+        <Card>
+          <CardItem
```
Comment (travisricks) [inline components/CreateChallenge/FormGroup02ChallengeType.tsx:123]: same comments as individual challenge page

File: components/CreateChallenge/FormGroup02ChallengeType.tsx [hunk 5]
```diff
@@ -119,6 +119,25 @@ function FormGroup02ChallengeType({navigation}, props) {
...
+            header
+            button
+            onPress={() => navigation.navigate('GroupChallengeLibrary')}>
+            <Text style={{paddingLeft: 120}}>Search Challenge</Text>
```
Comment (travisricks) [inline components/CreateChallenge/FormGroup02ChallengeType.tsx]: Please change to:
`Search Challenge Library`

File: components/CreateChallenge/FormGroup02ChallengeType.tsx [hunk 6]
```diff
@@ -119,6 +119,25 @@ function FormGroup02ChallengeType({navigation}, props) {
...
+          </CardItem>
+          <CardItem
+            button
+            onPress={() => navigation.navigate('GroupChallengeLibrary')}>
+            <Body>
+              <Text>
+                Pick challenge that someone created. Challenge title is
+                rewritten by the challenge you selected.
```
Comment (travisricks) [inline components/CreateChallenge/FormGroup02ChallengeType.tsx]: Please change to:
`Choose from a library of user-created challenges.`

File: components/CreateChallenge/FormGroup03ChallengeLibrary.tsx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+        </Content>
+      </Container>
+    );
+  } else {
+    return (
+      <Container style={styles.Container}>
+        <Content padder>
+          <Text>
+            Search challenge that you want to do and get started achieving your
+            goals.
```
Comment (travisricks) [inline components/CreateChallenge/FormGroup03ChallengeLibrary.tsx]: `Search through the challenge library to find one that will get you started achieving your goals.`

**Review by travisricks — CHANGES_REQUESTED** [review]
Looks good, @masumomo ! Just a few changes please

**Comment by travisricks** [comment]
Please make word changes on group and individual form pages

File: components/CreateChallenge/Form02ChallengeType.tsx (hunk 1 above)
Comment (masumomo) [reply components/CreateChallenge/Form02ChallengeType.tsx]: Wow I'm so sorry. by mistake!

**Review by travisricks — APPROVED** [review]
The changes look good!

**Pull request merged**